pkg/cluster/generateconfig.go

```diff
@@ -180,6 +180,7 @@ func (m *manager) generateInstallConfig(ctx context.Context) (*installconfig.Ins
 					ARO:                      true,
 				},
 			},
+			FIPS:       m.doc.OpenShiftCluster.Properties.ClusterProfile.FipsEnabled,
```

Comment:
Can customer change this post-creation? If so, then we probably need another [enricher](https://github.com/Azure/ARO-RP/blob/58ff96d63bebd42150d0dad4ebd23fa0f988323c/pkg/util/clusterdata/clusterdata.go#L43-L48).

Reply:
This value cannot be changed post creation. It is _technically_ possible, but not supported. 
> Enabling FIPS is a "day 1" operation, To enable or disable FIPS you need to uninstall and reinstall the cluster.
https://access.redhat.com/solutions/5594191